misc/benchmark/README.md

```diff
@@ -3,4 +3,9 @@ be found at the following address:
 
 https://github.com/ray-project/llmperf
 
-The code follows the standard Apache 2.0 license.
+The code follows the standard Apache 2.0 license. To run the benchmark, you need to login in with lepton CLI first via `lep login` command then use following command to launch the benchmark test:
+```
+python run.py --rounds 1 --concur-requests 50 --api-base "https://llama2-70b.lepton.run/api/v1/" --api-key `lep ws token` -m llama2-70b --max-tokens=100;
```

Comment:
I don't think we should hard-code the model url and api key. This works with any openai-compatible api bases, so we should write "for example"

```
python run.py \
  --rounds 2 \
  --concur-requests 4 \
  --api-base "OPENAI_COMPATIBLE_API_BASE" \
  --api-key API_KEY \
  --model MODEL_NAME \
  --max-tokens=100
```

If you are benchmarking lepton endpoints and you are logged in, you can use `lep ws token` to obtain the workspace token.